README.md

```diff
@@ -6,6 +6,62 @@ channel branches (e.g. `channels/8/edge`).
 Read more about the repository structure and build automation in [here](<https://github.com/ubuntu-rocks/.github/blob/main/profile/README.md#-joining-the-ubuntu-rocks-project>).
 
 
+## Image characteristics
+
```

Comment:
could you maybe add a few lines making the case for "why this comparison is being made?" , "why this temurin image in particular and not any other 3rd party jre?"

Reply:
I looked for images that are both popular and available and that's what I am coming up with:

According to Jetbrains 2022 survey of [Developer Ecosystem](https://www.jetbrains.com/lp/devecosystem-2022/java/) 60% of the developers still regularily used Java 8, 59% developed applications for Apache Tomcat and 67% used Spring Boot as an alternative to an application server.

New Relic 2022 [Java Ecosystem report](https://newrelic.com/resources/report/2022-state-of-java-ecosystem) listed following top 3 JDK vendors:
 - Oracle Corporation 34.48%
 - Amazon 22.04%
 - Eclipse Adoptimum 11.48%
 - Azul Zulu build of OpenJDK 16%

Snyk 2021 [JVM Ecosystem](https://snyk.io/jvm-ecosystem-report-2021/) had the following breakdown:
 - Eclipse Adoptimum 44%
 - Oracle Corporation OpenJDK 28%
 - Oracle Corporation JDK 23%
 - Azul Zulu build of OpenJDK 16%
 - Amazon Coretto 8%

The differences in the vendor distribution could be attributed to the audience providing survey responses.

The chiselled JRE container is built based on Ubuntu 22.04 version of Java 8 runtime - `8u352-b08`.

This section provides a comparison with readily-available JRE 8 images for Ubuntu 22.04:
 - Eclipse Adoptium: `eclipse-temurin:8u352-b08-jre-jammy`
 - Amazon: `amazoncorretto:8u352-alpine3.14-jre`

Azul Zulu does not provide a JRE image: https://hub.docker.com/r/azul/zulu-openjdk and it was not evaluated.
Oracle does not provide official image of the Java Runtime Environment 8 and it was not evaluated.

Reply:
The major points of difference with Corretto image are:
 - Corretto deploys busybox as a shell
 - Corretto does not have fontconfig/fonts libraries. This causes `Font.createFont()` fail with `java.lang.UnsatisfiedLinkError`.
 - Corretto does not provide `glibc` libraries imported by URL proxy selector.

Reply:
nice, thanks. I've added some extra typo fixes to the new text ;)